notifications_utils/version.py

```diff
@@ -1 +1 @@
-__version__ = '43.9.1'
+__version__ = '43.10.0'
```

Comment:
Removing the `bleed` property makes this a breaking change.

Reply:
```suggestion
__version__ = '44.0.0'
```